frontend/packages/dev-console/src/actions/modify-application.ts

```diff
@@ -41,3 +41,19 @@ export const EditApplication = (model: K8sKind, obj: K8sResourceKind): KebabOpti
     },
   };
 };
+
+export const EditHealthCheck = (model: K8sKind, obj: K8sResourceKind): KebabOption => {
+  return {
+    label: 'Edit Health Checks',
+    href: `/k8s/ns/${obj.metadata.namespace}/${obj.kind || model.kind}/${
```

Comment:
will it be possible `obj` has metadata/spec but no kind?

Reply:
Yes, it might be possible obj has no kind. Because in ```K8sResourceKind``` kind is an optional property.